content/specific-skill-success-criteria/basic-data-analysis/_index.md

```diff
@@ -5,4 +5,151 @@ title: 'Assessment: Basic data analysis'
+
+```
+Formula for standard deviation:  square root of [(1/N)*Sigma i=1 to N of (x - u)^2 ]
+sample_data = [9, 2, 5, 4, 12, 7, 8, 11]
+
+Step 1: Calculate the Mean (average value) = (9+2+5+4+12+7+8+11)/8 = 7.25
+
+Step 2:  For each number in sample_date, subtract the mean from the number and square the result
+
+(9-7.25)^2 = 3.06
+(2-7.25)^2 = 27.56
+(5-7.25)^2 = 5.06
+(4-7.25)^2 = 10.56
+(12-7.25)^2 = 22.56
+(7-7.25)^2 = 0.06
+(8-7.25)^2 = 0.56
+(11-7.25)^2 = 14.06
+
+[3.06, 27.56, 5.06, 10.56, 22.56, 0.06, 0.56, 14.06]
+
+Step 3:  Calculate the mean of all the numbers squared you calculated in step 2.  It is important to notice
+         that once you have calculated step 3 you already have the Variance, the calculation done in step 4
+         will simply give you the standard deviation.
+
+(3.06+27.56+5.06+10.56+22.56+0.06+0.56+14.06)/8 = 10.435 (This is the variance)
+
+Step 4:  Take the square root of step 3 and you are done
+
+Square root of 10.435 = 3.23
+```
+
+
+
+### Statistical concepts: Correlation and p-value (Probability value)
+A correlation coefficient is represented in Python by the variable R, the value ranges between -1 and 1.  A correlation coefficient of -1 (Negative slope) or +1 (Positive slope) illustrates a perfect linear relationship between dependant and independent variables, a correlation coefficient of 0 indicates NO linear relationship between dependant and independent variables.
+
+P-values, a.k.a. probability values are exactly that, if you wanted to know what the probability is that you would throw a 2 on a 6-sided die then you could calculate it like this: 2/6 = 0,333. So in English we would say you have a two in six chance of landing on a 6 and this equates to a 33,33% probability.
+
+Now suppose we have a hypothesis, we hypothesize that we have a fair coin, that is, no matter how much we flip this coin the probability of it landing on the one side is equal to it landing on the other side.  We shall call this hypothesis of ours the “null-hypothesis”.  If we were to reject our own hypothesis then we would have to have some evidence to do so and we would then also have to accept another hypothesis, this other hypothesis would be that the coin is not a fair coin, we shall call this other hypothesis the alternative hypothesis.  Lets summarize what we have below:
+```
+Null hypothesis:  We have a fair coin
+
+Alternative hypothesis: We don’t have a fair coin, it is rigged
+```
+
+
+Ok, now we start flipping this coin and we record what we land on, here is what we’ve found:
+```
+1st flip:  Heads – Chances of this happening is ½ = 50%
+2nd flip:  Heads – Chances of this happening is ¼ = 25%
+3rd flip:  Heads – Chances of this happening is 1/8 = 12,5%
+4th flip:  Heads – Chances of this happening is 1/16 = 6,25%
+.
+.
+.
+100th flip:  Heads – Chances of this happening is 1/2^n = 7,886 x 10^-31
```

Comment:
This is also stated in a confusing way. If I flip a coin 1000 times, and then I flip it once more then it is still a fair coin and it still has a 50% chance of hitting heads. The coin does not remember.

You are calculating the chances of getting 100 heads in a row. Not just a head on the 100th throw. 

In any case, how can we test that the learner understands it?

Reply:
The idea of the this example is to show how a p-value decreases rapidly the more unlikely the Null hypothesis becomes.  The chances of landing on heads 100 times in a row should be extremely unlikely and this is displayed by an extremely small p-value.  I wanted the candidates to make that connection:  extremely unlikely = extremely small p-value.
extremely likely = extremely high p-value.  I've added some tests to the 10 questions section which tests this.